python_codes/test_simulated_maps8.py

```diff
@@ -0,0 +1,273 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Thu Sep 27 14:27:26 2018
+7: reprise feb 2020
+Notably we try other pattern of borders (the generic ones) to make propensity maps.
+"""
+import scipy.ndimage
+import scipy.io as sio
+from pylab import *
+import random
+import itertools
+
+import scipy.stats as stats
+from scipy.stats import rv_discrete
+
+import glob
+import os
+import time
+
+import statsmodels.api as sm
+lowess = sm.nonparametric.lowess
+import matplotlib
+
+os.chdir("/home/axel/Bureau/z_python_scripts_copy/")
+import scn
+import ice_mirny3
+import distance_law_human
+import hicstuff as hcs
+
+# Experimental data:
+m=loadtxt("/home/axel/Bureau/YEAST/data_yeast_pds5/ALL_EXPERIMENTS_PDF/AT147_Pds5-AID-noTir1-G1-cdc20-TS_files/txt/MAT_RAW_chr5_AT147_Pds5-AID-noTir1-G1-cdc20-TS_2kb.txt")
+Nreads = sum(m)
+Nrealisations = 10
+
+#imshow(m**0.15,interpolation="none",cmap= 'afmhot_r')
+#colorbar()
+n1 = shape(m)[0]   # size of the chromosome contact map 
+
+# Normalisation:
+matscn1=ice_mirny3.ice_func(m,0)
+matscn1=scn.scn_func(matscn1,0)
+
+
+# Borders from experimental data:
+chrm=5
+b = loadtxt("/home/axel/Bureau/YEAST/data_yeast_pds5/borders_detection/borders_POISSIAN_Dplus_chr"+str(chrm)+"_.txt", dtype="int")
+b = list(b)
+ele_to_remove = [221,224,246,249]
```

Comment:
Where do these indices come from ?

Reply:
OK these are neighbours of empty lines, we need to recompute it from the input data

Reply:
Fixed in 786de98cbe6cae7937266e63400c8a2d9a7f3de7, `ele_to_remove` are now computed directly from the missing bins of the matrix.